Makefile

```diff
@@ -0,0 +1,21 @@
+
+.DEFAULT_GOAL := website
+
+postgres:
+	docker run -d --name=nnev-postgres -p 127.0.0.1:5432:5432 postgres | exit 0
+
+website: postgres
+	docker rm -f nnev-website | exit 0
+	docker build --force-rm -t nnev-website .
+	docker run --name=nnev-website --net=host -p 127.0.0.1:80:80 -v "$(shell pwd):/usr/src/" nnev-website
+
+stop:
+	docker rm -f nnev-website
+	docker kill postgres | exit 0
```

Comment:
evtl. fehlt hier auch `docker rmi …`

Reply:
nope, rmi ist nicht erforderlich.

Reply:
nicht erforderlich, aber verhindert, dass es dir die platte zuspammt?

Reply:
dafür ist cleanup da. `stop` ist nicht als das "ganz am Ende"-target gedacht, sondern wenn man aus irgendeinem Grund temporär abschalten will. `stop` soll explizit nicht die Images wegwerfen, damit man die nicht nochmal runterladen muss.

Reply:
und warum dann `rm` verwenden, wenn du nur stoppen willst dann ist vielleicht der Name irreführend?

Reply:
ok, rmi fliegt wieder und wird in ein fullcleanup-target migriert. Sonst muss man bei jedem Websitetest den ganzen Kram wieder komplett runterladen, was schon echt 'n bisschen nerven kann, weil das komplette Image zu bauen grad doch schon 'n weilchen dauert.